packages/expo-module-scripts/babel.config.node.js

```diff
@@ -0,0 +1,27 @@
+module.exports = () => ({
+  presets: [
+    [
+      require('@babel/preset-env'),
+      {
+        targets: {
+          node: '12.0.0',
```

Comment:
This seems low -- for SDK 50 I think we could set 18 to the minimum.

Reply:
Yes, I agree, but I didn't want to change anything of the babel preset when migrating it from `expo/expo-cli` to `expo/expo`. So for this PR, I'd like to keep this on Node 12.